base/card-api.gts

```diff
@@ -114,9 +110,43 @@ class ContainsMany<FieldT extends CardConstructor> implements Field<FieldT> {
     }
     return new WatchedArray(() => recompute(instance), value);
   }
+
+  component(model: Box<Card>, format: Format) {
+    let fieldName = this.name as keyof Card;
+    let field = this;
+    let arrayField = model.field(fieldName, useIndexBasedKey in this.card) as unknown as Box<Card[]>;
+    // TODO I'd like to construct an array of cards to use to render each item in the field (since 
+    // they can be polymorphic), however, i'm seeing that when I get the box's field value (the array 
+    // in question), that interferes with the component stability for the components that we construct
+    // for each item of the array. For example, just having model.value[fieldName] causes out component 
+    // stability test to fail. Perhaps there is a glimmer consideration we need to take into account here.
```

Comment:
@ef4 I'm gonna need some assistance here. I'm not to sure how to access  the underlying field's value (so that we can gather the polymorphic field cards) without inadvertently triggering component re-renders on arrays whose elements are supposedly stable.